k9mail-library/src/main/java/com/fsck/k9/mail/internet/MimeUtility.java

```diff
@@ -1153,4 +1155,12 @@ static boolean isFormatFlowed(String contentType) {
         }
         return false;
     }
+
+    static boolean isDelSp(String contentType){
```

Comment:
Well, that code probably shouldn't live in k9mail-library then. But we don't need to fix that now.

Please create a new class `FlowedMessageUtils` in the library module and put this method there instead of in `MimeUtility`.